client/packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -76,7 +77,7 @@ export function Autocomplete<T>({
   inputProps,
   ...restOfAutocompleteProps
 }: PropsWithChildren<AutocompleteProps<T>>): JSX.Element {
-  const filterOptions = createFilterOptions(filterOptionConfig);
+  const filterOption = filterOptions ?? createFilterOptions(filterOptionConfig);
```

Comment:
Wasn't sure what to rename this since I needed filterOptions for resorting the select component in programs 🤔